nilearn/plotting/surf_plotting.py

```diff
@@ -421,6 +422,106 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
+def _montage_display_mode_parser(display_mode):
+    """Checks whether the display_mode string passed to plot_surf_montage
+    is meaningful.
+
+    display_mode: str
+        Any combination of 'lateral', 'medial', 'dorsal', 'ventral',
+        'anterior', 'posterior', united by a '+'. Ex.: 'dorsal+lateral'
+    """
+
+    available_modes = {'lateral', 'medial',
+                       'dorsal', 'ventral',
+                       'anterior', 'posterior'}
+
+    desired_modes = display_mode.split('+')
+
+    if any(mode not in available_modes for mode in desired_modes):
+        wrong_modes = set(desired_modes).difference(available_modes)
+        raise ValueError('Unknown modes in display_mode: %s' % wrong_modes)
+
+    return desired_modes
+
+
+def plot_surf_montage(stat_map, meshes=None, mask_img=None,
+                      inflate=False, display_mode='lateral+medial',
+                      **kwargs):
+    """Convenience function to plot multiple views of plot_surf_stat_map
+    in a single figure. It projects STAT_MAP into MESHES and plots views of
+    left and right hemispheres. The DISPLAY_MODE argument defines the views
```

Comment:
Would you then have all the `left` or `right` stacked in a single column or display them in a single row?

Reply:
I'm not exactly sure about that. By default, I would stick to the conventions of pysurfer...

Reply:
I think you can stack them in a single column, because it is what you do when you have plot both hemispheres. it seems to be what pysurfer does, at least by default (but I'm not sure, I've never used it):

https://pysurfer.github.io/documentation/split_brain.html